codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -262,7 +262,7 @@ public int onDataRead(final ChannelHandlerContext ctx, int streamId, ByteBuf dat
                 }
 
                 if (endOfStream) {
-                    lifecycleManager.closeRemoteSide(stream, ctx.newSucceededFuture());
+                    lifecycleManager.closeStreamRemote(stream, ctx.newSucceededFuture());
```

Comment:
I'm not a big fan of the `Side` terminology here and the method name feel like it would be more appropriate as a method on `Http2Stream` rather than `Http2LifecycleManager`, but I can change back if want.